<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочник китайских мифических существ</string>
  </property>
  <widget class="QLabel" name="label_buttop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Чтобы посмотреть существо, вам нужно просмотерть таблицу и щелкнуть по выбраному существу и вам покажет доступную 
информацию про это существо</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>790</width>
     <height>710</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>631</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>⤓</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
